packages/xs-vat-worker/xs-lin.mk

```diff
@@ -0,0 +1,41 @@
+MODDABLE=$(PWD)/moddable
+TOOLS=$(MODDABLE)/build/bin/lin/release/
+NODE_MODULES=$(PWD)/node_modules
+ROOT=$(PWD)/../..
+
+$(NODE_MODULES)/.bin/xs-vat-worker: build/bin/lin/debug/xs-vat-worker
```

Comment:
I'm not sure I'd put `xs-vat-worker` in `node_modules/.bin/`, it's not particularly easy/useful to run by hand, so I don't think it deserves to be added to $PATH. The primary caller is code that can afford to import a library to find out the pathname. It might make this simpler if we don't try to copy it into place, but just use it directly from the compiler's output filename.

Reply:
I'm in the habit of blowing away the whole `build/` directory regularly, so I figured it was worth copying somewhere else.

But maybe keeping the `build/` directory around isn't that much trouble.

Reply:
good point. At some point we'll figure out a `make install` -type target for deployment, and at that stage we should do something more sophisticated than running it out of `build/`